### ISSUE TITLE
Uncaught TypeError: p.animatedPathSegList is not iterable 

### ISSUE THREAD

**Issue by texxnik** [proceduraljigsaw/Fractalpuzzlejs#2]
Hey! when uploading a file, everything works well in Firefox, but in Chrome it works with an error! 

Uncaught TypeError: p.animatedPathSegList is not iterable
    at FileReader.fr.onloadend ((index):1207)

Google Chrome
Version 95.0.4638.54

**Comment by proceduraljigsaw** [comment]
Yep, you're right. I found this about 15 minutes after release...

Turns out I'm using a deprecated interface to get the SVG data I need from the uploaded file. Chrome deprecated it years ago while firefox keeps it for some reason. 

I'm already porting it to the new interface, however there are lots of changes involved. I coded my own svg parser/rasterizer and it turns out that functionality is readily given by the new interface. I have it partially working already, but I can't dedicate much time right now.

So yes, it's firefox only for now, I'm sorry.
